packages/djangoflow_error_reporters/djangoflow_sentry_reporter/example/lib/src/error_reporting_service.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:djangoflow_error_reporter/djangoflow_error_reporter.dart';
+import 'package:djangoflow_sentry_reporter/djangoflow_sentry_reporter.dart';
+import 'package:flutter/foundation.dart';
+
+import 'env.dart';
+
+// Class to handle error reporting with DjangoflowSentryReporter
+class ErrorReportingService {
```

Comment:
I think we should keep the demo very simple and easy to understand. IMO, this extra class `ErrorReportingService` is not needed.